source/reduce/reducer.cpp

```diff
@@ -153,6 +168,25 @@ Reducer::ReductionResultStatus Reducer::Run(
   return Reducer::ReductionResultStatus::kComplete;
 }
 
+void Reducer::AddDefaultReductionPasses() {
```

Comment:
As explained in PR description, I have added this so tests (and other library consumers) can get a reducer with the default reduction passes. Otherwise, this code was only in `tool/reduce.cpp`

Reply:
Good idea.